binaries/runtime/src/operator/python.rs

```diff
@@ -137,21 +137,27 @@ pub fn spawn(
             };
             input.metadata.parameters.open_telemetry_context = Cow::Owned(string_cx);
 
-            let status_enum = Python::with_gil(|py| {
+            let status = Python::with_gil(|py| -> Result<i32> {
+                let pool = unsafe { py.new_pool() };
```

Comment:
We should add a comment here to clarify why this step is needed. For example, something like this:

```suggestion
                // We need to create a new pool here because the dora-runtime
                // is currently started through a `start_runtime` wrapper function,
                // which is annotated with `#[pyfunction]`. This attribute creates a
                // `GILPool` that lasts for the entire lifetime of the `dora-runtime`.
                // We want thatthe `PyBytes` created below are freed earlier, so
                // create a new `GILPool` here that is freed at the end of this block.
                // See https://github.com/PyO3/pyo3/pull/2864 and
                // https://github.com/PyO3/pyo3/issues/2853 for more details.
                let pool = unsafe { py.new_pool() };
```

Reply:
Indeed. Thank you!